core/src/main/java/com/orientechnologies/orient/core/storage/ridbag/sbtree/OSBTreeRidBag.java

```diff
@@ -619,21 +621,17 @@ public Object returnOriginalState(List<OMultiValueChangeEvent<OIdentifiable, OId
   }
 
   @Override
-  public int getSerializedSize() {
-    int result = 2 * OLongSerializer.LONG_SIZE + 3 * OIntegerSerializer.INT_SIZE;
-    if (ODatabaseRecordThreadLocal.instance().get().getStorage() instanceof OStorageProxy
-        || ORecordSerializationContext.getContext() == null)
-      result += getChangesSerializedSize();
-    return result;
+  public int getSerializedSize() {    
+    return 0;    
   }
 
-  @Override
-  public int getSerializedSize(byte[] stream, int offset) {
-    return getSerializedSize();
-  }
+//  @Override
```

Comment:
You can get rid of old methods, without commenting them. We have github for the history ;-)

Reply:
You are right :)